### PR TITLE
Fix inability to add comments inside mediators in synapse

### PR DESCRIPTION
**Pull request by dinuish94**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/synapse/config/xml/BeanMediatorFactory.java [hunk 1]
```diff
@@ -74,6 +74,8 @@ public Mediator createSpecificMediator(OMElement elem, Properties properties) {
             handleException("'action' attribute of Bean mediator is required.");
         }
 
+        addAllCommentChildrenToList(elem, mediator.getCommentsList());
+
```
Comment (chanikag) [inline modules/core/src/main/java/org/apache/synapse/config/xml/BeanMediatorFactory.java]: Remove these new lines

File: modules/core/src/main/java/org/apache/synapse/config/xml/SequenceMediatorFactory.java [hunk 2]
```diff
@@ -69,6 +69,7 @@ public SequenceMediator createAnonymousSequence(OMElement elem, Properties prope
         }
         addChildren(elem, seqMediator, properties);
         seqMediator.setSequenceType(SequenceType.ANON);
+
```
Comment (chanikag) [inline modules/core/src/main/java/org/apache/synapse/config/xml/SequenceMediatorFactory.java]: Unnecessary new line?

File: modules/core/src/main/java/org/apache/synapse/config/xml/BeanMediatorFactory.java (hunk 1 above)
Comment (dinuish94) [reply modules/core/src/main/java/org/apache/synapse/config/xml/BeanMediatorFactory.java]: Fixed in https://github.com/wso2/wso2-synapse/pull/1439

File: modules/core/src/main/java/org/apache/synapse/config/xml/SequenceMediatorFactory.java (hunk 2 above)
Comment (dinuish94) [reply modules/core/src/main/java/org/apache/synapse/config/xml/SequenceMediatorFactory.java]: Fixed in https://github.com/wso2/wso2-synapse/pull/1439

Comment (dinuish94) [reply modules/core/src/main/java/org/apache/synapse/config/xml/SequenceMediatorFactory.java]: Fixed in https://github.com/wso2/wso2-synapse/pull/1439

**Pull request merged**